### DISCUSSION TITLE
What does libraries with api support mean?

### DISCUSSION THREAD

**Discussion by ghost** [Nytuo/CosmicComics#13] [Q&A]
Does it mean it has auto scrobbling?

**Comment by Nytuo** [comment]
Hi,

You can create a library that depends on an API that will fetch information about the series / books in it. Depending of the API, it will fetch cover, description, characters, authors and related content like other pieces in the series.
Effectively, this is a little bit unclear on the description, I will correct-it soon.
Please note that some libraries needs an API Key to work out-of-the-box and since the wiki section is not completed there is no tutorial at this time.
